### PR TITLE
[wip] Added delete functionality for a user

### PR DESCRIPTION
**Pull request by sjainit**
Added a function to delete the listens of a user.

### CODE REVIEW

**Comment by sjainit** [comment]
@paramsingh, can we use `drop_user(username)` for the function https://github.com/metabrainz/listenbrainz-server/blob/e5e8590ee21a5495c5e240fad6e031e80bb5bbb9/listenbrainz/listenstore/influx_listenstore.py#L916 ?

File: listenbrainz/listenstore/influx_listenstore.py [hunk 1]
```diff
@@ -960,5 +960,28 @@ def query(self, query):
                 time.sleep(1)
 
 
+    def delete_listens_for_user(self, musicbrainz_id):
```
Comment (paramsingh) [inline listenbrainz/listenstore/influx_listenstore.py]: this function is not needed, we already have a delete function that does this.

**Comment by shivam-kapila** [comment]
@sjainit. Hi. I guess the intended task is related to adding a button in front end. Should I help you?

**Comment by paramsingh** [comment]
Closing in favor of #726

**Pull request closed without merging**